src/service/member-api-adapter/index.js

```diff
@@ -41,9 +41,19 @@ export default function MemberAPIService(config) {
         return serviceOptions;
     };
 
+    const generateUserQuery = function (params) {
+        if (Array.isArray(params.userId)) {
```

Comment:
I don't think you need to check if this is an array - you can simplify this with something like

```
if (!params.userId | !params.userId.length)  return ''; //should this throw an error instead?
const uidQuery = ([].concat(params.userId)).join('&userId=');
return `userId=${uidQuery}`

```